product_docs/docs/pgd/5/overview/index.mdx

```diff
@@ -24,59 +22,57 @@ Read about why PostgreSQL is better when it’s distributed with EDB Postgres Di
 
-Logical replication starts by copying a snapshot of the data from the source node. Once that's done, later commits are sent to other nodes as they occur in real time. Changes are replicated without executing SQL again, so the exact data written is replicated quickly and accurately.
+  - **Monitoring tools**: To monitor performance, health, and usage with PGD, you can utilize its [built-in command-line interface](../cli) (CLI), which offers several useful commands. For instance, the `pgd show-nodes` command provides a summary of all nodes in the cluster, including their state and status. The `pgd check-health` command checks the health of the cluster, reporting on node accessibility, replication slot health, and other critical metrics. The `pgd show-events` command lists significant events like background worker errors and node membership changes, which helps in tracking the operational status and issues within the cluster. Furthermore, the BDR extension allows for monitoring your cluster using SQL using the [`bdr.monitor`](../security/pgd-predefined-roles/#bdr_monitor) role.
 
-Nodes apply data in the order in which commits were made on the source node, ensuring transactional consistency is guaranteed for the changes from any single node. Changes from different nodes are applied independently of other nodes to ensure the rapid replication of changes.
+#### Node types and roles
 
-Replicated data is sent in binary form when it's safe to do so.
+  - **[Data nodes](../node_management/node_types/#data-nodes)**: Store and manage data, handle read and write operations, and participate in replication.
 
+  - **[Subscriber-only nodes](../node_management/subscriber_only/#subscriber-only-nodes)**: Subscribe to changes from data nodes for read-only purposes, used in reporting or analytics.
 
-### Connection management
+  - **[Witness nodes](../node_management/witness_nodes/)**: Participate in concensus proceses without storing data, aiding in achieving quorum and maintaining high availability.
 
-[Connection management](../routing) leverages consensus-driven quorum to determine the correct connection endpoint in a semi-exclusive manner to prevent unintended multi-node writes from an application. This approach reduces the potential for data conflicts. The node selected as the correct connection endpoint at any point in time is referred to as the [write leader](../terminology/#write-leader).
+  - **[Logical standby nodes](../node_management/logical_standby_nodes/)**: Act as standby nodes that can be promoted to data nodes if needed, ensuring high availability and disaster recovery.
 
-[PGD Proxy](../routing/proxy) is the tool for application connection management provided as part of EDB Postgres Distributed.
+  - **[Write leader](../terminology/#write-leader)**: Receives all write operations from PGD Proxy.
 
-### High availability
+### Architectural aims
```

Comment:
I kind of hate that we put this into HA topic, I much prefer us talking about flexibility, which high availability is just effect/result of.

I.e. flexibility in:
- rolling maintenance operations (blue/green)
   - postgres minor/major upgrades
   - system upgrades
   - schema changes
   - vacuum
- automated failover and self-healing (node coming back after failure/disconnect automatically reconnects to cluster and continues working)
- selective replication
- configurable durability (commit scopes)
- multiple write nodes
- read scaling with SO nodes

The text under mentions most of this, but it's all talking about HA which limits the scope of things IMO (i.e. it does not leave space for things like commit scopes).

Also, I would like to remove discussion about logical standbys as much as possible, we don't want to encourage use of those, if anything we want to discourage use of those.